Document/0x05b-Basic-Security_Testing.md

```diff
@@ -66,6 +66,62 @@ Once you've configured the network and established a connection between the test
 
 After completing these steps and starting the app, the requests should show up in the interception proxy.
 
+As mentioned before, starting at Android 7, Android device will no longer trust user CA certificates by default, unless specified in the application. In the following section, we explain two methods to bypass this android security mesure.
+
+#### Bypassing the Network Security Configuration
+
+From Android 7 onward, the network security configuration allows apps to customize their network security settings, by defining which CA certificates the app will be trusting. The CA certificates trusted by the app can be a system trusted CA as well as a user non-public CA.
+Network security configuration uses an XML file where the app specify which CAs certificates will be trusting.
```

Comment:
The network security configuration uses an XML file where the app specifies which CA certificates will be trusted.